src/server/ServerMethod.js

```diff
@@ -87,6 +87,7 @@ export default class ServerMethod {
             path: url,
             params: urlParams
           },
+          postBody: args,
```

Comment:
Hey, I wonder if it's maybe you are not sending an object??
On Wed, 26 Aug 2015 at 6:29 am Phil Cockfield <phil@cockfield.net> wrote:

> Hmm. This is getting weird. Can we do a screen share or pare on this. I'd
> like to see what's going on.
> On Tue, 25 Aug 2015 at 9:18 pm Nic Wise <notifications@github.com> wrote:
>
>> In src/server/ServerMethod.js
>> <https://github.com/philcockfield/rest-methods/pull/18#discussion_r37847197>
>> :
>>
>> > @@ -87,6 +87,7 @@ export default class ServerMethod {
>> >              path: url,
>> >              params: urlParams
>> >            },
>> > +          postBody: args,
>>
>> I was going with that originally, but it's not really an argument - its
>> the entire post body in this case. I'm happy with either one tho.
>>
>> I'm still totally not sure why this isn't being passed into the method in
>> my code in the first place...
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/philcockfield/rest-methods/pull/18/files#r37847197>.
>>
>